src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: hsl(234, 80%, 12%);
+  font-family: "Roboto", sans-serif;
+}
+
+.card__attribure {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__raiting,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  line-height: 18px;
+}
+
+.card__price {
+  color: hsl(234, 80%, 12%);
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: hsl(234, 80%, 12%);
+}
+
+.card__price-title {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
```

Comment:
Let's use #`color_code` in all places, cause now you use 2 approaches `hsl` and `#` - it's not okay